library/waku_thread/inter_thread_communication/requests/debug_node_request.nim

```diff
@@ -30,7 +30,7 @@ proc process*(
   of RETRIEVE_LISTENING_ADDRESSES:
     return ok($(%*waku.node.getMultiaddresses()))
   of RETRIEVE_MY_ENR:
-    return ok($(%*waku.node.enr.toURI()))
+    return ok(waku.node.enr.toURI())
```

Comment:
Maybe we need to do the same with the other cases

Reply:
There's only one additional case, `RETRIEVE_LISTENING_ADDRESSES`, and I think it's needed there because the result of `getMultiaddresses()`is a sequence, so we need to convert the sequence to a string